cmd/mycelo/genesis_commands.go

```diff
@@ -167,6 +167,15 @@ func envFromTemplate(ctx *cli.Context, workdir string) (*env.Environment, *genes
 		}
```

Comment:
Any reason not to add the CLI flag for `forks.gFork` too?

Reply:
Do you mean this below or something else?
https://github.com/celo-org/celo-blockchain/blob/f75b4a3b3f1068d4672b7c8791e4444cb4810331/cmd/mycelo/genesis_commands.go#L170-L178

Reply:
The flag definition was actually missing :)